custom_connectors/oauth2/clause.rb

```diff
@@ -0,0 +1,675 @@
+          'default to \'untitled\'.',
+          default: 'My New Workato Contract'
+        }, {
+          name: 'markdown', type: :string, optional: true,
+          hint: 'The contents of the contract in <a href="https://docs.accordproject.org/docs/markup-cicero'\
+          '.html" target="_blank">CiceroMark<a/> format.',
+          default: 'The *Acceptance Criteria* are the specifications defined...'
+        }, {
+          name: 'status', type: :string, optional: true,
+          hint: 'One of either `DRAFTING`, `SIGNING`, `RUNNING`, `COMPLETED`.'
+        }, {
+          name: 'createdAt', type: 'date_time',
+          hint: 'When contract drafting was started.',
+          optional: false
+        }, {
+          name: 'updatedAt', type: 'date_time', optional: false,
+          hint: 'When the contract was last updated.'
+        }]
+      end
+    },
+
+    #
+    # INPUT DEFINITIONS
+    #
+
+    contract_created_input: {
+      fields: lambda do
+        [
+          {
+            name: 'since',
+            type: :timestamp,
+            hint: 'When you start recipe for the first time, it picks up ' \
+            'trigger events from this specified date and time. Leave ' \
+            'empty to get events created one hour ago',
+            optional: true
+          },
+          {
+            name: 'organization',
+            type: :string,
+            control_type: 'select',
+            hint: 'The target organization to watch for new contracts',
+            pick_list: 'organizations',
+            optional: false
+          }
+        ]
+      end
+    },
+
+    contract_new_input: {
+      fields: lambda do
+        [{
+          name: 'organization',
+          type: :string,
+          control_type: 'select',
+          pick_list: 'organizations',
+          hint: 'The target organization that the new contract will be '\
+            'created in.',
+          optional: false
+        }, {
+          name: 'name',
+          type: :string,
+          hint: 'The name of the contract to be created. An empty name will '\
+            'default to \'untitled\'.',
+          default: 'My New Workato Contract',
+          optional: true
+        }, {
+          name: 'markdown',
+          type: :string,
+          hint: 'The contents of the new contract in <a '\
+            'href="https://docs.accordproject.org/docs/markup-cicero.html" '\
+            'target="_blank">CiceroMark<a/> format.',
+          default: 'The *Acceptance Criteria* are the specifications '\
+            'defined ...',
+          optional: false
+        }]
+      end
+    },
+
+    contract_select_input: {
+      fields: lambda { |_connection, _config_fields|
+        [].concat(
+          call('contract_field', 'The contact that you want to connect to.'),
+          call(
+            'organization_field',
+            'The target organization where the source contract exists.'
+          )
+        )
```

Comment:
Lets swap the two input fields around so that users know to give organization_field before contract field dropdown is displayed